app/src/pages/inside/stepPage/stepPage.jsx

```diff
@@ -46,7 +45,6 @@ import {
   editDefectsAction,
   linkIssueAction,
   postIssueAction,
-  isDefectGroupOperationAvailable,
```

Comment:
You can completely delete this utility if it is not used anywhere.

Reply:
It uses in component ActionPanelWithGroupOperations.